elephant/phase_analysis.py

```diff
@@ -307,3 +307,80 @@ def phase_difference(alpha, beta):
     delta = alpha - beta
     phase_diff = np.arctan2(np.sin(delta), np.cos(delta))
     return phase_diff
+
+
+def weighted_phase_lag_index(signal_i, signal_j, sampling_frequency,
+                             absolute_value=True):
+    r"""
+    Calculates the Weigthed Phase-Lag Index (WPLI)
+
+    This function expects the two signals (each containing multiple trials).
+    It Fourier-transforms the signals and for each trial pair, it calculates
+    the cross-spectrum. With that the WPLI is calculated.
+
+    Parameters
+    ----------
+    signal_i, signal_j: (t, n) np.array
```

Comment:
In the tests, it appears that Quantity is passed to the function. If the function supports multiple input types, then they must be listed here, and specific unit tests should be implemented for each type to check for any potential breaking changes when packages are updated. Moreover, the docstring should be formatted according to the conventions and style guide (e.g., like it was done with `phase_locking_value`).